feather.opam

```diff
@@ -31,3 +31,9 @@ build: [
     "@doc" {with-doc}
   ]
 ]
+depexts: [
+  # For pgrep
+  ["procps"] {os-family = "debian"}
+  ["procps-ng"] {os = "linux" & os-distribution = "archlinux"}
+  ["proctools"] {os = "macos" & os-distribution = "homebrew"}
```

Comment:
Do we actually need this for macos? I have pgrep installed in /usr/bin and it doesn't seem to be from homebrew? I don't have proctools installed, at least.

Reply:
Well, I'm not a mac user, so I googled "macos pgrep" and got this: https://unix.stackexchange.com/questions/225/pgrep-and-pkill-alternatives-on-mac-os-x

And I only read the first answer :)

I'll remove this.

Reply:
👍 thanks!